mumbleBot.py

```diff
@@ -314,6 +318,17 @@ def is_admin(self, user):
         else:
             return False
 
+    # =======================
+    #         Users changed
+    # =======================
+
+    def users_changed(self, user, message):
+        own_channel = self.mumble.channels[self.mumble.users.myself['channel_id']]
+        # if the bot is the only user left in the channel
+        if len(own_channel.get_users()) == 1: 
+            self.log.info('Other users in the channel left. Stopping music now.')
+            self.clear()
```

Comment:
https://github.com/azlux/botamusique/blob/master/mumbleBot.py#L528 maybe pause() should be better ?

Reply:
I like the idea. I prefer clear, since when everybody leaves, we like to start with a fresh music selection the next time. I'll leave it up to you as I am fine with both decisions.